COVID_19_model.ipynb

```diff
@@ -1229,11 +1336,11 @@
         "colab": {}
       },
       "source": [
-        "#@title Optionally download all html plots\n",
-        "download_plotly_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
+        "#@title Optionally download html EDA plots\n",
+        "download_data_plots = True #@param [\"False\", \"True\"] {type:\"raw\"}\n",
```

Comment:
Change this to False by default? It's a bit weird to have a Colab download 100 files by default as a non Intelecy user.

Reply:
Yeah definitely